tests/flow/create_test_rdb_file.py

```diff
@@ -35,7 +35,7 @@ def parse_timestamp(ts):
     return calendar.timegm(date_time_obj.timetuple()) * 1000
 
 def load_into_redis(redis_conn):
-    r = redis_conn.pipeline()
+    r = redis_conn.pipeline(transaction=False)
```

Comment:
@danni-m What is this for? (transaction=False)

Reply:
We use pipeline here for perf reasons we don't need the transactional pipeline which will fail on RLEC because its uses multi slots.